experiment/test_run_experiment.py

```diff
@@ -61,7 +61,21 @@ def setUp(self):
     def test_missing_required(self, mocked_error):
         """Tests that an error is logged when the config file is missing a
         required config parameter."""
-        # All but cloud_compute_zone.
+        # All but trials
+        del self.config['trials']
+        with mock.patch('common.yaml_utils.read') as mocked_read_yaml:
+            mocked_read_yaml.side_effect = lambda config_filename: self.config
```

Comment:
nit: You can just do `mocked_read_yaml.return_value = self.config`.

Reply:
`return_value` looks clearer in semantics than `side_effect`.
Just to confirm there will be no functionality loss for changing from `side_effect` to `return_value`. Is there any reason behind choosing `side_effect` at other similar test functions in https://github.com/google/fuzzbench/blob/master/experiment/test_run_experiment.py?

Reply:
In some cases, it makes sense to use side_effect, but I don't think there are any such cases in `test_run_experiment` I'll change the others now.

Reply:
https://github.com/google/fuzzbench/pull/409